influxdb_client/client/write_api_async.py

```diff
@@ -122,4 +122,4 @@ async def write(self, bucket: str, org: str = None,
                                                               precision=write_precision, async_req=False,
                                                               _return_http_data_only=False,
                                                               content_type="text/plain; charset=utf-8")
-        return response[1] == 204
+        return response[1] in (201, 204)
```

Comment:
Perhaps it's worth doing the same thing as the [Golang client](https://github.com/influxdata/influxdb-client-go/blob/52afd676229af18dc383dc7b7a340f2d8f562dbe/api/http/service.go#L115-L123) and treating any 2xx as a success? Either is good though 👍